.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:6.10
+      - image: circleci/node:8.11
```

Comment:
Maybe 8.10, same as in Lambda?

Reply:
Or leave 6.10 if middy wants to support 6.10 in Lambda for some time?

Reply:
I'm happy to change it to either, personally since I use the v.8.x line I'd like to ensure my code works on that line, not least because v.8.x uses a different es code that typescript is set to make it's compiled code compatible with.

@lmammino any thoughts on this? Is there someone else that feels very close to this topic that can provide some guidance?

Reply:
I agree, this should be 6.10 as we still support this version of Node.js

Reply:
Sweet, thank you @lmammino and @vladgolubev 

I'll make sure tsconfig targets the emcascript that 6.10 supports also.

Reply:
There's probably some way in circle CI to run the tests in both versions. If that's the case we should integrate v8.10 too. But I would consider this a separate issue